Packs/GSuiteAdmin/Integrations/GSuiteAdmin/GSuiteAdmin.py

```diff
@@ -1484,6 +1493,397 @@ def gsuite_chromeos_device_action_command(client: Client, args: dict[str, str])
+            else:
+                while len(cb_list_resp) < int(limit):
+                    if int(limit) - len(cb_list_resp) > API_LIMIT:
+                        page_size = str(API_LIMIT)
+                    else:
+                        page_size = str(int(limit) - len(cb_list_resp))
+                    params_for_command = assign_params_chromebrowser_list(projection, query, order_by, sort_order, org_unit_path,
+                                                                          page_token, page_size)
+                    response = client.http_request(full_url=full_url, method='GET', params=params_for_command)
+                    page_token = response.get('nextPageToken', '')
+                    cb_list_resp.extend(response['browsers'])
+                    if not page_token:
+                        break
+        else:
+            response = client.http_request(full_url=full_url, method='GET', params=params_for_command)
+            cb_list_resp.extend(response['browsers'])
+
+    readable_output = tableToMarkdown(HR_MESSAGES['CHROME_BROWSER_LIST'].format(device_id), cb_list_resp,
+                                      ['deviceId', 'osPlatform', 'osVersion', 'machineName', 'serialNumber', 'orgUnitPath'],
+                                      headerTransform=pascalToSpace, removeNull=True)
+    return CommandResults(outputs_prefix=OUTPUT_PREFIX['CHROME_BROWSERS'],
+                          outputs_key_field=['deviceId'],
+                          outputs=cb_list_resp,
+                          readable_output=readable_output,
+                          raw_response=response)
+
+
+def modify_policy_command(client: Client, args: dict[str, str]) -> str:
+    """
+        get a user details based on user key.
+
+        :param client: Client object.
+        :param args: Command arguments.
+
+        :return: String that confirms request was executed.
+    """
+    client.set_authorized_http(scopes=SCOPES['POLICY_MANAGEMENT'])
+    customer_id = args.get('customer_id', '')
+    target_type = args.get('target_type', '')
+    policy_raw_json = args.get('policy_raw_json', '')
+    policy_field_json_entry_id = args.get('policy_field_json_entry_id', '')
+    target_resource = args.get('target_resource', '')
+    additional_target_keys = args.get('additional_target_keys', '')
+    policy_schema = args.get('policy_schema', '')
+    policy_value = args.get('policy_value', '')
+    update_mask = args.get('update_mask', '')
+
+    if target_type == 'Group':
+        full_url = f'https://chromepolicy.googleapis.com/v1/customers/{customer_id}/policies/groups:batchModify'
+        target_resource_customized = f'groups/{target_resource}'
+    else:
+        full_url = f'https://chromepolicy.googleapis.com/v1/customers/{customer_id}/policies/orgunits:batchModify'
+        target_resource_customized = f'orgunits/{target_resource}'
+
+    atk_temp = '{' + additional_target_keys + '}'
+    atk_dict = json.loads(atk_temp)
```

Comment:
maybe it's better to let the user provide the whole json:
1. can avoid potential JSON decode errors for the user
2. Correct me if I'm wrong, but if I understand correctly the user can provide this argument as a JSON from the `gsuite-policy-resolve` command output, under the context key `GSuite.Policy.targetKey.additionalTargetKeys` (consider adding it to the command outputs in the yml)